tfx/components/example_gen/utils.py

```diff
@@ -272,6 +288,17 @@ def _verify_split_pattern_specs(
     raise ValueError('Only one %s is allowed in %s' %
                      (VERSION_SPEC, split.pattern))
 
+  if range_config:
+    if range_config.HasField('static_range'):
+      # For ExampleGen, RangeConfig must specify an exact span to look for,
+      # since only one span is processed at a time.
+      start_span_number = range_config.static_range.start_span_number
+      end_span_number = range_config.static_range.end_span_number
+      if start_span_number != end_span_number:
+        raise ValueError(
+            'Start and end span numbers for RangeConfig.static_range must '
+            'be equal: (%s, %s)' % (start_span_number, end_span_number))
+
```

Comment:
separate this to another validation function? this looks loose coupled with rest of the code in this util